.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+.code
```

Comment:
No need for these

Reply:
I and other IDE users would disagree with you there